src/model/wallet/SimpleWallet.ts

```diff
@@ -38,44 +37,41 @@ export class SimpleWallet extends Wallet {
      * @param encryptedPrivateKey
      */
     constructor(name: string,
-                network: NetworkType,
                 address: Address,
-                creationDate: LocalDateTime,
                 /**
-                 * The encrypted private key and information to decrypt it
+                 * The encrypted private key as Sjcl Cipher
                  */
-                public readonly encryptedPrivateKey: EncryptedPrivateKey) {
-        super(name, network, address, creationDate, 'simple_v1');
+                public readonly encryptedPrivateKey: SjclCipherEncrypted) {
+        super(name, address, 'simple_v2');
```

Comment:
Should the hardcoded version still be used here? this will make the sdk depends on the simple wallet.